### PR TITLE
maestral: add page

### PR DESCRIPTION
**Pull request by whitemuu**
new command

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/maestral.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# maestral
+
+> Maestral is a lightweight Dropbox client for macOS and Linux.
```
Comment (EmilySeville7cfg) [inline pages/common/maestral.md:3]:
```suggestion
> Maestral is a lightweight Dropbox client for macOS and Linux.
> More information: <https://maestral.app/cli>.
```

File: pages/common/maestral.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`maestral pause`
+
+- Resume syncing:
+
+`maestral resume`
+
+- Get sync status of a specific file or folder:
+
+`maestral filestatus {{local_path}}`
```
Comment (EmilySeville7cfg) [inline pages/common/maestral.md:23]:
```suggestion
`maestral filestatus {{path/to/file_or_directory}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/maestral.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# maestral
+
+> Maestral is a lightweight Dropbox client for macOS and Linux.
```
Comment (CleanMachine1) [inline pages/common/maestral.md:3]:
```suggestion
> A lightweight Dropbox client for macOS and Linux.
```

**Comment by CleanMachine1** [comment]
Thanks for your contribution! I have left a review, after I will approve :)

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/maestral.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# maestral
+
+> A lightweight Dropbox client for macOS and Linux.
+> More information: <https://maestral.app/cli>.
+
+- Start the GUI:
+
+`maestral gui`
+
+- Get current status of Maestral:
```
Comment (CleanMachine1) [inline pages/common/maestral.md:10]:
```suggestion
- Print current status of Maestral:
```

File: pages/common/maestral.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Pause syncing:
+
+`maestral pause`
+
+- Resume syncing:
+
+`maestral resume`
+
+- Get sync status of a specific file or folder:
```
Comment (EmilySeville7cfg) [inline pages/common/maestral.md:22]:
```suggestion
- Print sync status of a specific file or folder:
```

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Thanks for your contribution! ❤️

**Pull request merged**